packages/cyberstorm/src/components/Toast/Toast.module.css

```diff
@@ -0,0 +1,181 @@
+.root {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  box-shadow: 0 12px 24px 5px rgb(0 0 0 / 0.65);
+}
+
+.contentWrapper {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  border-left: 3px solid var(--feature-color);
```

Comment:
I'm not a/the designer but I wonder if the 1px difference is significant enough to warrant not using `--border-radius--2` or `--border-radius--4`?